pkg/docker/docker.go

```diff
@@ -163,6 +171,60 @@ func New(config *api.DockerConfig, auth docker.AuthConfiguration) (Docker, error
 	}, nil
 }
 
+// StartContainer starts an existing container.
+func (d *stiDocker) StartContainer(id string, hostConfig *docker.HostConfig) error {
+	return d.client.StartContainer(id, hostConfig)
+}
+
+// KillContainerOptions kills the container using SIGKILL.
+func (d *stiDocker) KillContainer(containerID string) error {
+	opts := docker.KillContainerOptions{ID: containerID}
+	return d.client.KillContainer(opts)
+}
+
+// dockerExec executes the command in specified container
+func (d *stiDocker) dockerExec(containerName string, command []string) error {
```

Comment:
name this dockerExecAsRoot or make it take a user name as a parameter.  we want to be very clear this is doing things as root.